internal/app/api/handler/index/index.go

```diff
@@ -1,25 +1,33 @@
 package indexhandler
 
 import (
+	"fmt"
 	"io"
 	"net/http"
 	"os"
-
+	"encoding/json"
+	"github.com/globalsign/mgo/bson"
 	"github.com/TheFlies/ofriends/internal/pkg/respond"
+	"github.com/TheFlies/ofriends/internal/app/types"
+	"github.com/TheFlies/ofriends/internal/app/friend"
+	"github.com/gorilla/mux"
 )
 
 type (
 	// Handler is index web handler
-	Handler struct{}
+	Handler struct{
+		repo friend.Repository
```

Comment:
Handler should use a service, not contact repo directly.

Reply:
fixed